r2/kv/store.go

```diff
@@ -38,3 +38,59 @@ type StoreConfig struct {
 func NewStore(cfg StoreConfig) r2.Store {
 	return &store{cfg: cfg}
 }
+
+func (s *store) FetchNamespaces() ([]string, int, error) {
+	return nil, 0, nil
```

Comment:
take/leave: (just a personal preference when stubbing) I like to `panic("not implemented")` instead of returning a default value. Makes it easier to track down issues as you're filling in code